### DISCUSSION TITLE
Menu bar: Alt+[key] accelerator -> Can it be changed?

### DISCUSSION THREAD

**Discussion by autoca** [freeplane/freeplane#701] [Help] [answered]
Hi all,

I just now noticed that in order to open (drop down) the File menu on the main menu by using only the keyboard, Freeplane wants me to press Alt+"l" (lower case L).

In actually all English programs that I know, it is Alt+"f" and this is kind of ingrained in both brain and muscle memory.
Does anyone know if this can be changed or do I have to fall back on an unwanted workaround with AutoHotkey or something?

MTIA and regards, a.

**Comment by macmarrum** [comment, marked as answer]
This is what I can see when I press `Alt`.

![2022-09-30_10,57,51](https://user-images.githubusercontent.com/7197131/193233818-3548f227-0ce5-4473-8135-bbcbfe2e3649.png)

For me it's `Alt+f` to access `File`.

What do you see when you press `Alt`?

**Reply by autoca** [reply]
Thx for checking it out, mac - may I call you mac?

Turns out I had a hotkey set to Alt+f and then FP changed the File menu acc to Alt+l.
Plus, after removing that hotkey I had to restart FP to snap back to "normal".

Regards, a.
